src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -1388,7 +1388,7 @@ internal static int FromHex(char digit)
                     : ((int)digit - (int)'a'))
                     + 10);
             }
-            throw new ArgumentException(nameof(digit));
+            throw new ArgumentException(SR.Argument_NotHexDigit, nameof(digit));
```

Comment:
One derives from the other, and we're usually very lenient about giving back more derived instances. Obviously it could be a problem if someone compares the exact type, but catch blocks and casts will all "just work". Why the concern here? I mean, it's fine not to change it, I'm just curious about the hesitancy. Much bigger changes have been made elsewhere.